terraform/environment/ai_search.tf

```
@@ -0,0 +1,14 @@
+resource "azurerm_resource_group" "az_search_rg" {
+  name     = "${local.resource_prefix}-rg-search"
+  location = var.region
+  tags     = local.tags
+}
+
+resource "azurerm_search_service" "search" {
+  name                = "${local.resource_prefix}-az-search"
+  resource_group_name = azurerm_resource_group.az_search_rg.name
+  location            = azurerm_resource_group.az_search_rg.location
+  sku                 = var.ai_search_sku
+  replica_count       = var.ai_search_replica_count
+  partition_count     = var.ai_search_partition_count
```

Can you make sure to add tags to the AI search resource please? I've been using them to make sure all resources are tagged with the environment name (which helps map resources to an environment, since the actual resource names contain a hash, to avoid issues in name lengths).

```suggestion
  partition_count     = var.ai_search_partition_count
  tags                = local.tags
```